src/commands/signup.ts

```diff
@@ -133,7 +133,7 @@ async function collectName(): Promise<string | undefined> {
   const { name } = await inquirer.prompt([
     {
       name: "name",
-      message: "What is your full name?",
+      message: "What is your first and last name?",
```

Comment:
I assumed this was first and last name but wanted to drop a comment here to make sure.